src/storage/storage_manager_helpers.h

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef MXNET_STORAGE_STORAGE_MANAGER_HELPERS_H_
+#define MXNET_STORAGE_STORAGE_MANAGER_HELPERS_H_
+
+#if MXNET_USE_CUDA
+#include <cuda_runtime.h>
+#include "../common/cuda_utils.h"
+typedef  mxnet::common::cuda::DeviceStore CudaDeviceStore;
+#endif  // MXNET_USE_CUDA
+
+#include <thread>
+#include <sys/sysctl.h>
+#ifndef _WIN32
+#if __APPLE__
+#include <mach/vm_statistics.h>
+#include <mach/mach_types.h>
+#include <mach/mach_init.h>
+#include <mach/mach_host.h>
+#else
+#include <sys/sysinfo.h>
+#include <sys/mman.h>
+#include <sys/fcntl.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/resource.h>
+#endif  // __APPLE__
+#else
+#include <Windows.h>
+#include <process.h>
+#endif  // _WIN32
+
+#include <unordered_map>
+#include <vector>
+#include <tuple>
+#include <atomic>
+#include <iostream>
+#include <mutex>
+#include <new>
+#include <string>
+#include <limits>
```

Comment:
[include order needs fix](http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fsanity/detail/PR-18582/4/pipeline#step-80-log-238):
`[2020-06-19T17:55:27.961Z] src/storage/storage_manager_helpers.h:30:  Found C system header after C++ system header. Should be: storage_manager_helpers.h, c system, c++ system, other.  [build/include_order] [4]`

Reply:
You can test it locally by running `make cpplint`